server/index.js

```diff
@@ -7,25 +7,79 @@ const images = require('../database/config.js');
 const creds = require('../mongo_creds.js');
 
 const app = express();
-
-mongoose.connect(`mongodb://${creds.user}:${creds.pass}@54.173.228.201:27017/tpt`, { useNewUrlParser: true, useUnifiedTopology: true })
+//mongodb://${creds.user}:${creds.pass}@54.173.228.201:27017/tpt`
+mongoose.connect(`mongodb://localhost/tpt`, { useNewUrlParser: true, useUnifiedTopology: true });
 
 app.use(cors());
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 app.get('/:id/images', (req, res) => {
-  const name = req.params.id;
-  const search = {};
-  search[name] = /.*/;
-  images.find(search, name, (err, pics) => {
-    if (err) {
-      console.log(err);
-    } else {
-      res.send(pics[0]);
-    }
-  });
+	const name = req.params.id;
+	const search = {};
+	search[name] = /.*/;
+	images.find({ item: name }, (err, pics) => {
+		if (err) {
+			console.log(err);
+		} else {
+			res.send(pics[0]);
+		}
+	});
+});
+
+app.delete('/:id/images', (req, res) => {
```

Comment:
Your endpoints go to /images but you are performing CRUD on products, not images, right? A little misleading

Reply:
LG!